Links.html

```diff
@@ -1,15 +1,34 @@
 <!DOCTYPE html>
 <html>
-<ul>
-    <li><a href="index.html">Home</a></li>
-    <li><a href="Glossary.html">Glossary</a></li>
-    <li><a href="Links.html">Links</a></li>
-<h1>Useful Links</h1>
-
-<ul><a href="http://validator.w3.org/nu/">HTML Validator</a></ul>
-
-<hr></hr>
-<blockquote>You can make anything by writing
-<br><b>&mdash; C.S. Lewis</b></br></blockquote>
-<p>&copy; 2019<a href="mailto:Alison.Nadjiwon@student.saultcollege.ca?subject=Message from your CSD120 website"> Alison Nadjiwon</a></p>
+<head>
+    <title>Alison's CSD120 Website</title>
+    <meta charset="utf-8">
+</head>
+<body>
+
+    <header>
+        <nav>
+            <ul>
+                <li><a href="index.html">Home</a></li>
+                <li><a href="glossary.html">Glossary</a></li>
+                <li><a href="links.html">Links</a></li>
+            </ul>
+        </nav>
+    </header>
+
+    <main>
+        <h1>Useful Links</h1>
+
```

Comment:
You need a list item element (`<li>`) for each element in a list.
You were also missing the `target="_blank"` attribute that makes a hyperlink open in a new tab